shared/feature/profile/src/main/java/com/epmedu/animeal/profile/domain/LogOutUseCase.kt

```diff
@@ -2,20 +2,16 @@ package com.epmedu.animeal.profile.domain
 
 import com.epmedu.animeal.auth.AuthRequestHandler
 import com.epmedu.animeal.profile.data.repository.ProfileRepository
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.launch
 
 class LogOutUseCase(private val repository: ProfileRepository) {
-    private val coroutineScope = CoroutineScope(Dispatchers.IO)
 
     suspend operator fun invoke(
         onSuccess: () -> Unit,
         onError: (exception: Exception) -> Unit
     ) {
         val requestHandler = object : AuthRequestHandler {
             override fun onSuccess(result: Any?) {
-                clearProfileAfterLogOut(onSuccess)
```

Comment:
Do you clear profile after logout from Account page?

Reply:
Indeed, forgot to update it there, thank you!